packages/cli/src/cmds/stats/stats.ts

```diff
@@ -0,0 +1,258 @@
+                name: name,
+                elapsedInstrumentationTimeTotal: 0,
+                callNoReturnFound: true,
+                numberOfCalls: 0,
+              }
+            } else {
+              functionExecutionTimes[fileName].functions[functionKey].callNoReturnFound = true;
+            }
+          } else if (event.event == 'return') {
+            // A return's parent_id matches the call's id
+            let functionKey = 'thread_' + event.thread_id + '_id_' + event.parent_id;
+            if (functionKey in functionExecutionTimes[fileName].functions &&
+                functionExecutionTimes[fileName].functions[functionKey].callNoReturnFound) {
+              functionExecutionTimes[fileName].functions[functionKey].callNoReturnFound = false;
+              if ('elapsed_instrumentation' in event) {
+                functionExecutionTimes[fileName].functions[functionKey].elapsedInstrumentationTimeTotal += event.elapsed_instrumentation;
+              }
+              functionExecutionTimes[fileName].functions[functionKey].numberOfCalls += 1;
+            } // else something went wrong: found return with no call
+          }
+        });
+      });
+      verbose(v);
+
+      // now that all functions were collected, index them by function
+      // name, not by <thread_id,parent_id>
+      let totalFunctionExecutionTimes = {}
+      for (var fileName in functionExecutionTimes) {
+        for (var functionKey in functionExecutionTimes[fileName].functions) {
+          let name = functionExecutionTimes[fileName].functions[functionKey].name;
+          if (!(name in totalFunctionExecutionTimes)) {
+            totalFunctionExecutionTimes[name] = {
+              elapsedInstrumentationTimeTotal: 0,
+              numberOfCalls: 0,
+            }
+          }
+          totalFunctionExecutionTimes[name].elapsedInstrumentationTimeTotal += functionExecutionTimes[fileName].functions[functionKey].elapsedInstrumentationTimeTotal;
+          totalFunctionExecutionTimes[name].numberOfCalls += 1;
+        }
+      }
+
+      // convert hash to array
+      let flatFunctionExecutionTimes: {name: string,
+                                       elapsedInstrumentationTimeTotal: number,
+                                       numberOfCalls: number}[] = [];
+      for (var name in totalFunctionExecutionTimes) {
+        flatFunctionExecutionTimes.push({
+          name: name,
+          elapsedInstrumentationTimeTotal: totalFunctionExecutionTimes[name].elapsedInstrumentationTimeTotal,
+          numberOfCalls: totalFunctionExecutionTimes[name].numberOfCalls,
+        });
+      }
+
+
+      return flatFunctionExecutionTimes;
+    }
+
+    async function sortExecutionTimes(functionExecutionTimes) {
+      // sort the array
+      return functionExecutionTimes.sort((a, b) => b.elapsedInstrumentationTimeTotal - a.elapsedInstrumentationTimeTotal);
+    }
+
+    async function showStats() {
```

Comment:
This function doesn't have a declared type for the parameter or the return value